### ISSUE TITLE
How to add the row numbers? 

### ISSUE THREAD

**Issue by ganesh-s05** [fbaligand/kibana-enhanced-table#143]
Hi @fbaligand 

I had updated my plugin to the latest version. However, I could not find out on how to utilize the Row Number to my existing Enhanced Table. 

Could you please provide some documents/insights on how this can be achieved? 

******

Good news: enhanced table v1.10.0 is out with 'Add row number column' option feature and more enhancements!
https://github.com/fbaligand/kibana-enhanced-table/releases/tag/v1.10.0

_Originally posted by @fbaligand in https://github.com/fbaligand/kibana-enhanced-table/issues/76#issuecomment-703332978_

**Comment by fbaligand** [comment]
Hi @ganesh-s05,

There is a new checkbox in "Options" tab, with label "Add row number column".

![image](https://user-images.githubusercontent.com/3750300/95060391-80f40b00-06fa-11eb-8692-f45049fc80f7.png)

**Comment by ganesh-s05** [comment]
Hi @fbaligand , 

Unfortunately, i do not see it. 

![image](https://user-images.githubusercontent.com/67759874/95079300-0b0c9580-0734-11eb-86b3-207b453b6eb3.png)

I updated the plugin by first removing & then updating it. 
![image](https://user-images.githubusercontent.com/67759874/95079579-75253a80-0734-11eb-9274-020f454f1e36.png)

After which, i cleared my browser cache & then launched the Kibana dashboard. However, I do not see it. I tried new Visualization too with no luck. 
Am i missing something here?
